src/app/tests/suites/TestSaveAs.yaml

```diff
@@ -675,3 +675,71 @@ tests:
       attribute: "vendor_id"
       response:
           value: readAttributeVendorIdDefaultValue
+
```

Comment:
Shouldn't we have some tests where we are saving a nonzero-length string?

And where we verify that when we write the saveAs value the following read produces the thing we expect (without comparing to the saveAs value itself)?

Reply:
I have updated the tests as suggested.